### PR TITLE
Add a "Whirldwind Tour" to the guide

### PR DESCRIPTION
**Pull request by Technius**
This is a WIP PR to add a "Whirlwind Tour" to the guide, similar to the brief introduction in the current guide on the wiki.

### CODE REVIEW

**Comment by Technius** [comment]
I've been occupied with other things recently so any help here would be welcome.

**Review by rolandlo — COMMENTED** [review]
In general I pretty much like the whirldwind tour. Pdf annotation and export to pdf/svg/png should definitively be featured more prominently though. While describing how to create different shapes (which is also important), please do also briefly mention the shape recognizer. 
Don't hesitate to add more pictures, as that will help the novice to find things and make the reading more comfortable.

I agree with the overview, giving the user the choice to take the whirldwind tour or to look for more specific topics.

**Comment by Technius** [comment]
I updated the whirlwind tour just now. I pruned out much of the detail that I originally planned to include, as the whirlwind guide should be a sort of 5-10 minute tutorial that demonstrates bare minimum functionality.

I would like to see this merged sooner rather than later. It should not take very much effort to amend the guide in the future.

**Review by rolandlo — COMMENTED** [review]
Looks nice to me. Good job with the icons. I only have minor comments.

File: docs/guide/whirlwind-tour.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+* If you are using a combined wireless USB mouse/keyboard device, you may need
+  to switch the input device from `Mouse` or `Keyboard` to `Mouse+Keyboard
+  Combo`.
+* A drawing tablet stylus may be detected as a mouse, which means that pressure
+  sensitivity will not work. To fix this, you may need to override the device as
+  a `Stylus`.
+
+### Temporary tool button bindings
+
+The primary button of each input device will activate the currently selected
```
[the remark this answers is not in the document]
Comment (Technius) [reply docs/guide/whirlwind-tour.md]: I've clarified this section.

**Comment by Technius** [comment]
Thanks for the feedback. I inserted more icons and added the LaTeX tool to the tool table.

**Review by rolandlo — APPROVED** [review]
LGTM. Please go ahead and merge.

**Pull request merged**